includes/polling/functions.inc.php

```diff
@@ -564,3 +564,37 @@ function get_device_oid_limit($device)
         return 10;
     }
 }
+
+/**
+ * Update the application status and output in the database.
+ *
+ * @param array $device
```

Comment:
Good spot, I've made more use of it by adding it to the dbUpdate call where it should have been included anyway.